cowprotocol/solver_dashboard/protocol_weekly_rewards_3668034.sql

```diff
@@ -0,0 +1,11 @@
+-- This query computes all the rewards distributed to the solvers per week
+-- The rewards are split into 3 categories: performance, consistency and quote
+select
+    week_start,
+    sum(performance_reward) as performance_reward,
+    sum(consistency_reward) as consistency_reward,
+    sum(quote_reward) as quote_reward,
+    sum(performance_reward + consistency_reward + coalesce(quote_reward, 0)) as total_reward
+from query_3641173
```

Comment:
Same comment as above. If the query you reference is not part of this repo, please add it as well.

Reply:
Same issue, same answer. I'll add the query